django/db/migrations/state.py

```diff
@@ -443,6 +443,11 @@ def from_model(cls, model, exclude_rels=False):
                 elif name == "index_together":
                     it = model._meta.original_attrs["index_together"]
                     options[name] = set(normalize_together(it))
+                elif name == "indexes":
+                    indexes = model._meta.indexes
```

Comment:
I added a commit which should address the issue, if I understand it correctly. I think `deepcopy` is required instead of `[:]` which shallow copies.